alexandria/core/utils.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+
+from alexandria.core import models
+
+
+def create_document_file(user, group, document_attributes, file_attributes):
+    document = models.Document.objects.create(
+        created_by_user=user,
+        created_by_group=group,
+        modified_by_user=user,
+        modified_by_group=group,
+        **document_attributes,
+    )
+    file = create_file(document, user, group, file_attributes)
+
+    return document, file
+
+
+def create_file(document, user, group, attributes):
```

Comment:
Same here, please add a docstring

Reply:
Add reST docs or was the idea something different